src/snowflake/cli/api/config.py

```diff
@@ -373,6 +382,11 @@ def _bool_or_unknown(value):
     return {k: _bool_or_unknown(v) for k, v in flags.items()}
 
 
+def _read_config_file_toml() -> dict:
+    with open(CONFIG_MANAGER.file_path, "r") as f:
+        return tomlkit.loads(f.read()).unwrap()
```

Comment:
```suggestion
    return tomlkit.loads(CONFIG_MANAGER.file_path.read_text())
```

Reply:
Done - for this file and few lines later for connections.toml :)